jdaviz/configs/imviz/tests/test_orientation.py

```diff
@@ -152,3 +156,48 @@ def test_custom_orientation(self):
         lc_plugin._obj.add_orientation(rotation_angle=None, east_left=True, label=None,
                                        set_on_create=True, wrt_data=None)
         assert self.viewer.state.reference_data.label == "CCW 42.00 deg (E-left)"
+
+
+class TestDeleteOrientation(BaseImviz_WCS_WCS):
+
+    @pytest.mark.parametrize("klass", [EllipseSkyRegion, RectangleSkyRegion])
+    @pytest.mark.parametrize(
+        ("angle", "sbst_theta"),
+        [(0.5 * u.rad, 2.641593),
+         (-0.5 * u.rad, 3.641589)])

```

Comment:
Is -0.5 rad a good input for "negative angle"? If not, feel free to adjust the test.